packages/web/src/pages/sign-up-page/pages/SelectGenresPage.tsx

```diff
@@ -85,6 +88,24 @@ export const SelectGenresPage = () => {
                       value={value}
                       size='large'
                       type='checkbox'
+                      onClick={(e) => {
+                        if ((e.target as HTMLInputElement).checked) {
+                          dispatch(
+                            make(Name.CREATE_ACCOUNT_SELECT_GENRE, {
+                              genre: label,
+                              selectedGenres: [...currentGenres, label]
```

Comment:
maybe create the new selected genres first, and use in the dispatch and setCurrent?

Reply:
do we want to do this here, or maybe just during onSubmit?

Reply:
> do we want to do this here, or maybe just during onSubmit?
I'm thinking here so we know what genres they select as they go in case they drop off between this and artists page, and also to give us an indication that they're interacting with this page